scripts/bash/downloadEnvironmentVariables.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set +x # do not output commands in this script, as some would show credentials in plain text
+
+BRANCH=$1
+DIR=$(dirname "$0")
+
+# can provide one or more packages as command line arguments, or will default to all deployable
+# packages
+if [ -z $2 ]; then
+    echo "Fetching environment variables for all deployable packages"
+    PACKAGES=$(${DIR}/getDeployablePackages.sh)
+else
+    PACKAGES=${@:2}
+    echo "Fetching environment variables for ${PACKAGES}"
+fi
+
+echo ${LASTPASS_PASSWORD} | LPASS_DISABLE_PINENTRY=1 lpass login ${LASTPASS_EMAIL}
+
+for PACKAGE in $PACKAGES; do
+    ENV_FILE_PATH=${DIR}/../../packages/${PACKAGE}/.env
+
+    # checkout branch specific env vars, or dev as fallback, temporarily redirecting stderr
+    exec 3> /dev/stderr 2> /dev/null
+    lpass show --notes ${PACKAGE}.${BRANCH}.env > ${ENV_FILE_PATH} || lpass show --notes ${PACKAGE}.dev.env > ${ENV_FILE_PATH}
```

Comment:
Minor: I suggest a new group in LastPass to keep these separate from normal tupaia credentials.

Reply:
Already how it's done ;-)

rego@beyondessential.com.au _only_ has access to "Tupaia Environment Variables", which is separate from our main shared folder